resource/graph_node.go

```diff
@@ -28,11 +28,13 @@ type GraphNode struct {
 	markedForRemoval          bool
 	unresolvedDependencies    []string
 	needsDependencyResolution bool
+	timesReconfigured         uint8
 }
 
 var (
-	errNotInitalized  = errors.New("resource not initialized yet")
-	errPendingRemoval = errors.New("resource is pending removal")
+	maxReconfigAttempts uint8 = 5
```

Comment:
Can you export this and access it (instead of hardcoding `5`) from `graph_node_test` so if we change this value the test will automatically update?